src/content/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/rest-api-alerts/rest-api-calls-alerts.mdx

```diff
@@ -742,7 +742,7 @@ These API functions include links to the API Explorer, where you can create, upd
     id="create-nrql-conditions"
     title="Create NRQL conditions for policies"
   >
-    **Recommendation:** Due to the way NRQL data is streamed, set the `aggregation_method` to `EVENT_FLOW` and use the default settings of `1` for `aggregation_window` and `2` for `aggregation_timer`. `EVENT_FLOW` works in most use-cases, but for a discussion on which use cases work better with `EVENT_TIMER`, see [Which aggregation method to use?](https://discuss.newrelic.com/t/relic-solution-how-can-i-figure-out-which-aggregation-method-to-use/164288).
+    **Recommendation:** Due to the way NRQL data is streamed, set the `aggregation_method` to `EVENT_FLOW` and use the default settings of `1` for `aggregation_window` and `2` for `aggregation_delay`. `EVENT_FLOW` works in most use-cases, but for a discussion on which use cases work better with `EVENT_TIMER`, see [Which aggregation method to use?](https://discuss.newrelic.com/t/relic-solution-how-can-i-figure-out-which-aggregation-method-to-use/164288).
```

Comment:
In addition, both values are in seconds, in the API. This means that, for `aggregation_window`, the default setting is `60`, and for `aggregation_delay`, the default setting is `120`.